<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>864</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1310</width>
    <height>864</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1310</width>
    <height>864</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3DViewer 1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>451</width>
      <height>801</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="LFileName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>File Name: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="LNumberV">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Vertices: </string>
     </property>
    </widget>
    <widget class="QLabel" name="LNumberE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Edges: </string>
     </property>
    </widget>
    <widget class="QGroupBox" name="scalingGB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Scaling</string>
     </property>
     <widget class="QDoubleSpinBox" name="scaleSB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="movingGB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>431</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Moving</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="moveX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>y</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="moveY">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>z</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="moveZ">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="rotatingGB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>431</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Rotating</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="rotateX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>y</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="rotateY">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>z</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="rotateZ">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="projectionGB">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Projection</string>
     </property>
     <widget class="QRadioButton" name="pParallel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Parallel</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="pCentral">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Central</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="edgesGB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>431</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Edges</string>
     </property>
     <widget class="QPushButton" name="BBackgroundE">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Color</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="eDashed">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dashed</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="eSolid">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Solid</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Thickness</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="eWidth">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="verticesGB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>431</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Vertices</string>
     </property>
     <widget class="QPushButton" name="BBackgroundV">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Color</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="vSquare">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Square</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="vCircle">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Circle</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="vSize">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QRadioButton" name="vNone">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Background</string>
     </property>
     <widget class="QPushButton" name="BBackground">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Color</string>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="OGLWidget" name="openGLWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>801</width>
      <height>801</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>820</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>16</width>
      <height>801</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>16</width>
      <height>801</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1310</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave_as_Picture"/>
    <addaction name="separator"/>
    <addaction name="actionExport_to_GIF"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSave_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Settings"/>
    <addaction name="action_4"/>
    <addaction name="separator"/>
    <addaction name="actionReset_Settings"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_as_Picture">
   <property name="text">
    <string>Save as Picture</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExport_to_GIF">
   <property name="text">
    <string>Export to GIF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSave_Settings">
   <property name="text">
    <string>Save Settings</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad_Settings">
   <property name="text">
    <string>Load Settings</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+L</string>
   </property>
  </action>
  <action name="actionReset_Settings">
   <property name="text">
    <string>Reset Settings</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+R</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Apply Settings</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+A</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">oglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
